src/index.html

```diff
@@ -7,8 +7,37 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
+  <body class="body">
+    <header class="header">
+      <img
+      class="logo"
+      src="images/logo.png"
+      alt="moyo"
+      herf="/src"
+      >
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item"><a class="link link1 is-active" href="/apple">Apple</a></li>
```

Comment:
The class names 'link1', 'link2', etc. are not descriptive. It's better to give meaningful names to your classes that describe their function or content.